src/components/TripCard.spec.js

```diff
@@ -1,16 +1,55 @@
 import TripCard from './TripCard.js';
 import { render, screen } from '@testing-library/react';
+import { userEvent } from '@storybook/testing-library';
```

Comment:
I think you are not importing the right thing here. The following should be imported.
`import userEvent from '@testing-library/user-event';`

Reply:
Surprisingly the test fails if I don't import it with brackets 😅